integrated_channels/moodle/client.py

```diff
@@ -150,6 +151,100 @@ def _get_access_token(self):
                 response.status_code
             )
 
+    @moodle_request_wrapper
+    def _get_enrolled_users(self, course_id):
+        """
+        placeholder
+        """
+        params = {
+            'wstoken': self.token,
+            'wsfunction': 'core_enrol_get_enrolled_users',
+            'courseid': course_id,
+            'moodlewsrestformat': 'json'
+        }
+        user_response = requests.get(
+            urljoin(
+                self.enterprise_configuration.moodle_base_url,
+                self.MOODLE_API_PATH,
+            ),
+            params=params
+        )
+        return user_response
+
+    def get_creds_of_user_in_course(self, course_id, user_email):
+        """
+        placeholder
+        """
+        response = self._get_enrolled_users(course_id)
+        parsed_response = response.json()
+        user_id = None
```

Comment:
I don't believe that
```
if not user_id:
``` 
will appropriately handle the condition that we never find the user's email in the list of enrolled students without the assignment to None. However, please correct me if I'm wrong! Happy to remove it if it's unnecessary.

Reply:
I've read that first sentence several times and I don't quite understand.
You have: 
```
if enrollment.get('email') == user_email:
    user_id = enrollment.get('id')
    break
```
so if we never get a user_id from this loop because there is never a match then the next logic:
`if not user_id` would still catch it because user_id is still None. It's never defined. This is what I mean by not needing to set it explicitly to None.

Reply:
Resolved offline! I believe this comes down to preferences of implementation. There are ways around the predefining of `user_id` to None, but I think my solution is perfectly reasonable here.

Reply:
This was also my misunderstanding. I have used `if not var` in a lot of places where the variable was not defined in the given code block, but didn't think about the fact that function parameters still count as definition.